crates/test_utils/src/connector_auth.rs

```diff
@@ -159,33 +159,28 @@ impl ConnectorAuthenticationMap {
                             table.get("key1"),
                             table.get("api_secret"),
                             table.get("key2"),
+                            table.get("adyen"),
+                            table.get("wise"),
```

Comment:
Can we please keep the code as connector agnostic as possible and make as much changes as possible within the collection itself.

Ideally, we would want to modify this only if we want to add yet another key type that takes 5 more keys.